app/main.py

```diff
@@ -1 +1,60 @@
-#  write your code here
+class Animal:
+
+    list_of_animals = []
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+        Animal.list_of_animals.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = not self.is_hungry
+            food_points_holder = self.appetite
+            self.appetite = 0
```

Comment:
we don't have to chage appetite of animal, we should change only if its hungry or not